dynamic-simulation/dynamic-simulation-dsl/src/main/java/com/powsybl/dynamicsimulation/groovy/DynamicSimulationReports.java

```diff
@@ -16,26 +16,26 @@
 public final class DynamicSimulationReports {
 
     public static ReportNode buildRootDynamicSimulationTool() {
-        return ReportNode.newRootReportNode()
-                .withLocaleMessageTemplate("core.dynasim.dynamicSimulationTool", ReportBundleBaseName.BUNDLE_BASE_NAME)
+        return ReportNode.newRootReportNode(PowsyblCoreReportResourceBundles.BASE_NAME)
```

Comment:
A remark that might be helpful for the design of this API: you almost never use this part of the API in tests but prefer calling  `withMessageTemplateProvider` after

Reply:
I changed it in unit tests, they all use the `newRootReportNode(String... bundleBaseNames)` interface now. The withMessageTemplateProvider is kept for custom/expert usecases.